diff --git a/lib/chess.rb b/lib/chess.rb
index 49d6731..f229d0b 100644
--- a/lib/chess.rb
+++ b/lib/chess.rb
@@ -49,7 +49,7 @@ module ChessGame
         four: [nil, nil, nil, nil, nil, nil, nil, nil],
         three: [nil, nil, nil, nil, nil, nil, nil, nil],
         two: %w[P P P P P P P P],
-        one: %w[R N B K Q B N R],
+        one: %w[R N B K Q B N R]
       }
     end
   end
@@ -58,14 +58,14 @@ module ChessGame
   class Player
     include Pieces
 
-    def initialize
+    def initialize(side)
       @pawns = []
-      8.times { @pawns.push(Pawn.new([0, 1])) }
-      @bishops = Bishop.new([0, 1]), Bishop.new([0, 1])
-      @knights = Knight.new([0, 1]), Knight.new([0, 1])
-      @rooks = Rook.new(), Rook.new()
-      @queen = Queen.new()
-      @king = King.new()
+      8.times { @pawns.push(Pawn.new(side)) }
+      @bishops = Bishop.new(side), Bishop.new(side)
+      @knights = Knight.new(side), Knight.new(side)
+      @rooks = Rook.new(side), Rook.new(side)
+      @queen = Queen.new(side)
+      @king = King.new(side)
       @pieces = [@pawns, @bishops, @knights, @rooks, @queen, @king].flatten
     end
 
